public/app/features/admin/partials/users.html

```diff
@@ -55,7 +55,9 @@
             </a>
           </td>
           <td class="text-right">
-            <span class="label label-tag" ng-class="{'muted': user.isDisabled}" ng-if="ctrl.getAuthModule(user) === 'ldap'">LDAP</span>
+            <span class="label label-tag" ng-class="{'muted': user.isDisabled}" ng-style="ctrl.getUserAuthLabelStyle(user)" ng-if="ctrl.getAuthLabel(user)">
```

Comment:
using functions in angular binding is something we have been very strict on not doing (as it's a bad practice, can cause bad performance). 

But this view is going to be rewritten to react soon so maybe not a big deal. But this change adds 3 functions in bind expressions, is there a way to avoid that?

Reply:
Yea, that's dirty implementation, just because I know that should be rewritten in React. I'll try to rid of that, don't like it as well.